src/Model/SearchData.php

```diff
@@ -275,6 +287,59 @@ public function setSelectedCreator(?string $selectedCreator): SearchData
         return $this;
     }
 
+    /**
+     * @return array|null
+     */
+    public function getStatuses(): ?array
+    {
+        return $this->statuses;
+    }
+
+
+    /**
+     * @param array|null $statuses
+     * @return SearchData
+     */
+    public function setStatuses(?array $statuses): SearchData
+    {
+        $this->statuses = $statuses;
+        return $this;
+    }
+
+    public function addStatus(array $statuses): SearchData
+    {
+        foreach ($statuses as $name => $count) {
+            $this->statuses[$name] = $count;
+        }
+
+        return $this;
+    }
+
```

Comment:
Wird `addStatus()` wirklich benötigt, oder reicht nicht `addStatuses()`. Falls `addStatus()` separat benötigt wird, müsste sie anders implementiert werden (einen einzelnen Status-Wert als Parameter nehmen und keinen Array).
```suggestion
    public function addStatus(array $statuses): SearchData
    {
        foreach ($statuses as $name => $count) {
            $this->statuses[$name] = $count;
        }

        return $this;
    }

```